coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +238,38 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        try:
+            class_padding = cls.ClassPadding(
+                *(int(padding) for padding in tuple(
```

Comment:
I see why it's like that https://github.com/coala/coala/blob/41e4752a40530ec915fa01582bc73708e00c26b0/coalib/settings/Setting.py#L121

`Setting` offers common data type conversions. It has `list_delimeters` for list/tuple conversion. That's why we are getting `('1', '0')` while doing `tuple(docstyle_settings['class_padding'])` :tada: